modules/exploits/unix/http/pfsense_graph_injection_exec.rb

```diff
@@ -33,18 +34,27 @@ def initialize(info = {})
             [ 'URL', 'http://www.security-assessment.com/files/documents/advisory/pfsenseAdvisory.pdf']
           ],
         'License'        => MSF_LICENSE,
-        'Privileged'     => true,
+        'Platform'       => 'php',
+        'Privileged'     => 'true',
         'DefaultOptions' =>
           {
-            'SSL'         => true,
-            'Encoder'     => 'php/base64',
-            'PAYLOAD'     => 'php/meterpreter/reverse_tcp',
+            'SSL'     => true,
+            'PAYLOAD' => 'php/meterpreter/reverse_tcp',
+            'Encoder' => 'php/base64'
           },
-        'DisclosureDate' => 'Apr 18, 2016',
-        'Platform'       => 'php',
-        'Arch'           => ARCH_PHP,
-        'Targets'        => [[ 'Automatic Target', { }]],
+        'Arch'           => [ ARCH_PHP ],
+        'Payload'        =>
+          {
+            'Space'  => 6000,
+            'Compat' =>
+              {
+                'Arch'           => 'php',
```

Comment:
if you get rid of this line, the meterpreters show back up.  I think thats the way to go at this point and with that last change i'll be happy to land.